docs/source/scripts.rst

```diff
@@ -58,6 +58,40 @@ Fisher’s exact tests of each event.
 The output file contains the original columns with an additional column
 containing the p-values appended.
 
+annotate_aaseq_with_uniprot
+========================
+.. code:: sh
+
+    usage: annotate_aaseq_with_uniprot.py [-h] -i INPUT_AASEQ -r REFERENCE_SEQ -o OUTPUT
```

Comment:
remote .py from program name, these should be console programes create by pypi add will not have the .py extension